boards/shields/xen_dom0/doc/index.rst

```diff
@@ -0,0 +1,52 @@
+.. _xen_dom0:
+
+Xen Dom0: universal shield for XEN Dom-0
+##############################################
+
+Overview
+********
+
+This virtual shield allows user to build Zephyr as a Xen initial domain (Dom0). The feature
```

Comment:
The community does not like "virtual" shields, instead snippets should be used, even if they do not offer the same functionality. https://docs.zephyrproject.org/latest/build/snippets/index.html

Reply:
Just for clarification, when we were developing this functionality, the `snippets` concept didn't exist.
Thank you for pointing that out; we'll investigate this approach.